vnext/Universal.UnitTests/App.xaml

```diff
@@ -1,8 +1,7 @@
-﻿<Application
+<Application
     x:Class="ReactUWPUnitTests.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:ReactUWPUnitTests"
-    RequestedTheme="Light">
```

Comment:
Intentional? .. I think this was added to ensure the UI was more consistent on everyones box -- at least until we have theming more sorted.